src/main/java/org/cdc/gov/sdp/HL7Terser.java

```diff
@@ -4,31 +4,36 @@
 import org.apache.camel.Predicate;
 import org.apache.camel.RuntimeCamelException;
 import org.apache.camel.builder.ValueBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class HL7Terser {
+	private static final Logger LOG = LoggerFactory.getLogger(HL7Terser.class);
 
 	private enum HL7Operation {
 		EQUALS, CONTAINS, STARTS_WITH, MINIMUM, MAXIMUM
 
 	}
-	
+
 	private enum LogicalOperation {
 		AND, OR
 	}
 
 	private static final int MAX_REP = 100;
-	
+
 	public boolean filter(Exchange exchange) {
 		String filter = (String) exchange.getIn().getHeader("HL7Filter");
-		
-		return processTerm(filter.trim(), exchange);
+		boolean passed = processTerm(filter.trim(), exchange);
+
+		LOG.info("Exchange " + (passed ? "PASSED" : "FAILED") + " the filter:" + filter);
```

Comment:
I don't think this message is going to be meaningful as it is, since it doesn't include any detail about the message itself.